frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-root-page/feature-flag-root-page-content/feature-flag-root-section-card/feature-flag-root-section-card-table/feature-flag-root-section-card-table.component.ts

```diff
@@ -7,11 +7,12 @@ import {
   FeatureFlag,
 } from '../../../../../../../../core/feature-flags/store/feature-flags.model';
 import { MatTableDataSource, MatTableModule } from '@angular/material/table';
-import { AsyncPipe, NgIf, NgFor, UpperCasePipe } from '@angular/common';
+import { AsyncPipe, NgIf, NgFor, UpperCasePipe, DatePipe } from '@angular/common';
 import { MatTooltipModule } from '@angular/material/tooltip';
 import { TranslateModule } from '@ngx-translate/core';
 import { MatChipsModule } from '@angular/material/chips';
 import { RouterModule } from '@angular/router';
+import { CommonStatusIndicatorChipComponent } from '../../../../../../../../shared-standalone-component-lib/components';
```

Comment:
Is there an easy way to get rid of these confusing and brittle relative import paths? I tried a few things with path aliases in ts-config, but couldn't find anything simple that worked.

Reply:
yes there should be, maybe we can spawn a quick task for that.